Project2015To2017/ProjectConverter.cs

```diff
@@ -18,13 +18,24 @@ public sealed class ProjectConverter
 		private readonly ConversionOptions conversionOptions;
 		private readonly ProjectReader projectReader;
 
-		private static IReadOnlyCollection<ITransformation> TransformationsToApply(ConversionOptions conversionOptions)
+		private static IReadOnlyCollection<ITransformation>
+			TransformationsToApply(ConversionOptions conversionOptions, bool modernProject)
 		{
+			var targetFrameworkTransformation = new TargetFrameworkTransformation(
+				conversionOptions.TargetFrameworks,
+				conversionOptions.AppendTargetFrameworkToOutputPath);
+
+			if (modernProject)
+			{
+				return new ITransformation[]
+				{
+					targetFrameworkTransformation
```

Comment:
Could we keep some of the other transforms? They might apply if somebody had used an older version of the converter that didn't have all of the current simplifications. Not needed for this release though

Reply:
That's a matter for different PR. We shouldn't change CPS project files at all when we read and write them, unless any additional transformations are passed in `ConversionOptions`.